evalml/model_understanding/graphs.py

```diff
@@ -507,8 +507,26 @@ def _calculate_axis_range(arr):
     return [min_value - margins, max_value + margins]
 
 
-def _get_prediction_vs_actual_data(y_true, y_pred, outlier_threshold):
-    """Helper method to help calculate the y_true and y_pred dataframe, with a column for outliers"""
+def get_prediction_vs_actual_data(y_true, y_pred, outlier_threshold=None):
+    """Combines y_true and y_pred into a single dataframe and adds a column for outliers. Used in `graph_prediction_vs_actual()`.
+
+    Arguments:
+        y_true (pd.Series): The real target values of the data
+        y_pred (pd.Series): The predicted values outputted by the regression model.
```

Comment:
Looks like from impl that woodwork is supported too? Could you update docstr / tests? Otherwise I can wait for this to be merged and add it to a PR of mine too hehe

Reply:
Yep I can add it to this PR!

Reply:
Sweet, thank you!!